TaskSharper.Calender.WPF/Views/CalendarWeekView.xaml

```diff
@@ -48,7 +43,7 @@
                     </DataTemplate>
                 </ItemsControl.ItemTemplate>
             </ItemsControl>
-            <StackPanel Orientation="Vertical" Grid.Row="1" Grid.Column="2">
+            <StackPanel Orientation="Vertical" Grid.Column="2">
```

Comment:
Does Grid.Row get defaulted to 0 if it's left out?

Reply:
Added. Done